e2e/infrastructure/transaction/CreateTransactionFromDTO.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('CreateTransactionFromDTO', () => {
                     // tslint:disable-next-line:max-line-length
                     signature: '7442156D839A3AC900BC0299E8701ECDABA674DCF91283223450953B005DE72C538EA54236F5E089530074CE78067CD3325CF53750B9118154C08B20A5CDC00D',
                     signer: '2FC3872A792933617D70E02AFF8FBDE152821A0DF0CA5FB04CB56FC3D21C8863',
-                    version: 36867,
+                    version: 36865,
```

Comment:
We should not use this static number for versioning of transactions ;

Since this number has been taken over from nem2-library project, we should now be using constants to represent it. That includes the possibility changing the TransactionVersion class to permit converting `1` to its corresponding hexadecimal representation (0x9001).

Reply:
adding to it ; those transaction versions only work for MIJIN_TEST network types.

This is how you build the transaction version :

0x90 is network identifier (Mijin_Test)
0x01 is transaction version (1)

Reply:
OK in unit tests

Reply:
OK in unit tests.